packages/pwa-kit-react-sdk/src/config/schema.json

```diff
@@ -0,0 +1,17 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "pwa-kit-config-schema",
+    "title": "PWA Kit Config Schema",
+    "type": "object",
+    "required": ["app"],
+    "properties": {
+        "app": {
+            "$id": "#pwa-kit-config-schema/app",
+            "title": "App",
+            "type": "object",
+            "required": [],
+            "properties": {}
+        }
+    }
+}
```

Comment:
To jog my memory, in this pwa-kit "app" schema, we'd put the configuration schema for things that are configurable on the sdk level, say like the proxy config and potentially localization in the future, correct?

I do have a comment/opinion on the file name and location tho, I feel like it's not super flexible, the file name "schema" on it's own doesn't tell you much with out the parents name being "config". I'd recommend something like this:

```
-- schemas
  |
  -- app-config.json
  -- package.json
```

This folder allows for expandability if we want to have others schema's for others things (I put package.json in there because maybe we could at the same time, define what the proxy's look like in that file if we wanted to do that).

This is just a thought to open up discussion.